examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,22 +21,34 @@
      "name": "stdout",
```

Comment:
<p>I would remove this cell, seeing values for <code>accept</code>  doesn't provide any info, the plot alone is enough</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>